tools/iree-prof-output-utils.cc

```diff
@@ -175,6 +176,25 @@ const char* GetZoneName(const tracy::Worker& worker,
   return worker.GetZoneName(worker.GetSourceLocation(source_location_id));
 }
 
+std::string GetSourceFileLine(const tracy::Worker& worker,
+                              int16_t source_location_id) {
+  const auto& zone = worker.GetSourceLocation(source_location_id);
+  absl::string_view file_name = worker.GetString(zone.file);
+  if (file_name.empty() || file_name == "-") {
+    return "";
+  }
+  return absl::StrCat(file_name, ":", zone.line);
+}
+
+const tracy::PlotData* GetMemoryPlotData(const tracy::Worker& worker) {
```

Comment:
Do you know if we can get the memory data per dispatch?

Reply:
Maybe since there are quite detailed information about memory allocation. But, it requires some time to investigate what information is actually available by iree code.